TDD_Training/TDD_Training/FizzBuzzService.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+
+namespace TDD_Training
+{
+    public class FizzBuzzService
+    {
+        public string isMultipleOf(int givenNumber)
```

Comment:
Ideally, this method is not checking for a multiple of correct?
it is returning the Fizz or Buzz based on your input.
So I'd rename something like
CheckFizzBuzz or something.